includes/ziti/socket.h

```diff
@@ -63,7 +63,7 @@ ziti_context Ziti_load_context(const char *identity);
  * @return native socket handle
  */
 ZITI_FUNC
-ziti_socket_t Ziti_socket();
+ziti_socket_t Ziti_socket(int type);
```

Comment:
would you add info for what 'type' means? is it TCP/UDP? or raw or something else?

Reply:
updated